src/ostorlab/apis/assets.py

```diff
@@ -129,7 +129,6 @@ def __get_asset_variables(self) -> Dict[str, Dict]:
         elif isinstance(self._asset, link.Link):
             asset_type_variables = {"url": {"urls": [self._asset.url]}}
         elif isinstance(self._asset, ios_store.IOSStore):
```

Comment:
Why remove the TODO?

Reply:
we discussed that the change is difficult because we have many places that need to change package_name to bundle_id.